core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/AzureBatchSource.java

```diff
@@ -32,26 +32,28 @@
 @Name("AzureBlobStore")
 @Description("Batch source to read from Azure Blob Storage.")
 public class AzureBatchSource extends FileBatchSource {
-
   @SuppressWarnings("unused")
   private final AzureBatchConfig config;
 
   public AzureBatchSource(AzureBatchConfig config) {
-    super(config);
+    super(new FileBatchConfig(config.referenceName, config.fileRegex, config.timeTable, config.inputFormatClass,
+                              config.fileSystemProperties, config.maxSplitSize, config.ignoreNonExistingFolders,
+                              config.recursive, config.path, config.pathField, config.filenameOnly, config.schema));
     this.config = config;
   }
 
   /**
    * Plugin config for {@link AzureBatchSource}.
    */
-  public static class AzureBatchConfig extends FileBatchConfig {
-    @Description("The Microsoft Azure Storage account to use.")
+  public static class AzureBatchConfig extends FileSourceConfig {
+    @Description("Path to file(s) to be read. If a directory is specified,terminate the path name with a '/'. " +
+      "The path must start with `wasb://` or `wasbs://`.")
```

Comment:
include an example of what's expected. Should also validate this at configure time.